### DISCUSSION TITLE
Visual Studio crashes when opening solution (.sln) file

### DISCUSSION THREAD

**Discussion by MicEsp22** [microsoft/winget-cli#1034] [General]
Dear All,

I know this question/problem might sound very awful, or even stupid.

I am not an advanced programmer I first say (probably not even A programmer). But, I try to get better day after day, trying to learn as much as I can. So I thought that studying this real Big project, would have gave me the possibility to understand many, many things (with time).

However, running VS 2019 Community, as soon as I open the solution file, after (apparently) everything has been loaded, VS crashes without any particular message.

Maybe some of you might know the reason. I would be really grateful if you tell me what would be the issue.

Many thanks, and keep going!

**Comment by Josh65-2201** [comment]
Don't think this is a winget issue since it installs and opens. You should put this on VS feedback area.

**Reply by MicEsp22** [reply]
@josh-65 sorry I don't get what you mean with "it installs". The issue is that as soon as I open the main .sln file, after some seconds VS closes itself. So I'm not able do actually do anything. I was only able to open a .ccp file, but well, it did not prevent VS to close. 
Do you think is a VS problem? Could be, I was maybe thinking that the version required is not provided from the actual version of VS I have installed locally. Could it be? Thanks

**Comment by denelon** [comment]
@MicEsp22
There are a few others who have attempted opening the Windows Package Manager project in Visual Studio who have encountered trouble with the unit tests. You could do some searches in closed issues, and other discussions to get some additional hints.

**Reply by MicEsp22** [reply]
@denelon thanks for the reply. I will go have a look at those, hoping to find the solution to this issue. Will keep you updated!

**Reply by denelon** [reply]
@JohnMcPMS do we already have guidance for the supported versions of Visual Studio?

**Reply by JohnMcPMS** [reply]
I'm not sure, but I would agree with above comments that if VS Community will attempt to open the solution, it shouldn't crash.

We use VS Enterprise 2019 internally, but I don't think it should be restricted to that.  I don't even know what the difference in all of the VS SKUs is.

**Reply by MicEsp22** [reply]
Could it be that some VS extension are not installed ( for example, I see a .vdproj - visual deployment project ?? ) ?

I was thinking that the source could have been a versioning incompatibility, but it turns out that I am working on a .vfproj which solution has quite all the same versioning settings of this .sln file. So, I would say that is not the problem.

**Reply by MicEsp22** [reply]
@denelon @JohnMcPMS apparently, I finally made it not to crash anymore. 

Well, it is time to build before to "cheer". 
Anyway, the issue was related to the Intel Advisor 1.0 extension. So, I don't know how much this has to do strictly with You ( meaning the project itself ), either can I understand why it was then making VS auto-closing right after opening Your .sln file.
